metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1605,6 +1604,13 @@ class MetalsLanguageServer(
         disconnectOldBuildServer().asJavaObject
       case ServerCommands.DecodeFile(uri) =>
         fileDecoderProvider.decodedFileContents(uri).asJavaObject
+      case ServerCommands.ChooseClass(params) =>
+        fileDecoderProvider
+          .chooseClassFromFile(
+            params.textDocument.getUri().toAbsolutePath,
+            params.includeInnerClasses
```

Comment:
Having to declare this in params here leaks the logic of understanding tasty files to the editor, which we should avoid.